JASP-Desktop/components/JASP/Widgets/RectangularButton.qml

```diff
@@ -99,12 +99,13 @@ Item
 			width:	Math.min(filterButtonRoot.width - (2 * buttonPadding), height)
 			height: filterButtonRoot.height - (2 * buttonPadding)
 
-			sourceSize.width:	Math.max(96, width  * 2)
-			sourceSize.height:	Math.max(96, height * 2)
+		//	sourceSize.width:	Math.max(96, width  * 2)
```

Comment:
I did not see a difference between specifying the sourcesize or not...

Reply:
The duplicate feature should be available also from the result.

Reply:
okido